be2-scala/src/test/scala/ch/epfl/pop/storage/DbActorSuite.scala

```diff
@@ -493,6 +493,58 @@ class DbActorSuite extends TestKit(ActorSystem("DbActorSuiteActorSystem")) with
     list should contain(message2)
   }
 
+  test("CreateLao messages can be written and read back in catchup on lao channel") {
+    // arrange
+    val initialStorage: InMemoryStorage = InMemoryStorage()
+    val dbActor: AskableActorRef = system.actorOf(Props(DbActor(mediatorRef, MessageRegistry(), initialStorage)))
+    val channel = Channel(CHANNEL_NAME)
+
+    // Write some message using WriteCreateLaoMessage
+    val writeAsk = dbActor ? DbActor.WriteCreateLaoMessage(channel, MESSAGE)
+    val writeAnswer = Await.result(writeAsk, duration)
```

Comment:
Since sending a WriteCreateLaoMessage can throw a DbActorNackException, maybe you could replace calls to Await.result by Await.ready in order to query the dbActor in a cleaner way. See : https://stackoverflow.com/questions/41170280/difference-await-ready-and-await-result

Reply:
I am not convinced, this is a unit test meant to pass, hence no errors are expected. The point is that the test failed and something need fixing. I don't see the point of handling errors since it will end the same way: Test failed